etc/kayobe/inventory/group_vars/all/dev-tools

```diff
@@ -0,0 +1,8 @@
+---
+########################################################################
+# List of development tool packages to install during host configuration
+dev_tools_packages_extra:
+  - tcpdump
```

Comment:
`tcpdump` is already part of `dev_tools_packages_default`.

Reply:
removed